pyproject.toml

```
@@ -33,15 +33,15 @@ classifiers = [
 ]
 requires-python = ">= 3.11, < 4"
 dependencies = [
-  "frequenz-api-common >= 0.6.0, < 0.7.0",
+  "frequenz-api-common >= 0.8.0, < 0.9.0",
   "googleapis-common-protos >= 1.56.4, < 2",
   # We can't widen beyond 6 because of protobuf cross-version runtime guarantees
```

Damn, too many mentions of version numbers.


```suggestion
  # We can't widen beyond 8 because of protobuf cross-version runtime guarantees
```